velox/functions/prestosql/tests/ElementAtTest.cpp

```diff
@@ -67,9 +67,8 @@ class ElementAtTest : public FunctionBaseTest {
       const std::vector<VectorPtr>& parameters) {
     auto result =
         evaluate<SimpleVector<T>>(expression, makeRowVector(parameters));
-    if (result->size() != 1) {
-      throw std::invalid_argument(
-          "elementAtSimple expects a single output row.");
+    if (result->size() < 1) {
+      throw std::invalid_argument("elementAtSimple expects non-empty output.");
```

Comment:
So `elementAtSimple()` now could produce a result of more than one rows, but still only return the first row? This may be confusing to other folks in the future. Could we create a separate version for evaluation of more than one rows (or change the test to only produce one row)?